EVENT.md

```diff
@@ -581,9 +581,13 @@ Unlike other filter keys whose values must be a list of strings, the
 `_lookback_window` value must be a positive integer that represents a positive
 duration in seconds.
 
-When available on a filter, the duration since the source was registered must be
-less than or equal to the parsed lookback window duration for the filter to
-match.
+When available on a filter (in `filters`), the duration since the source was
+registered must be less than or equal to the parsed lookback window duration for
+the filter to match, i.e., it must be inside the lookback window.
+
+When available on a negated filter (in `not_filters`), the duration since the
```

Comment:
```suggestion
When present on a filter (in `filters`), the duration since the source was
registered must be less than or equal to the parsed lookback window duration for
the filter to match, i.e., it must be inside the lookback window.

When present on a negated filter (in `not_filters`), the duration since the
```